bigtable-dataflow-parent/bigtable-dataflow-import/src/test/java/com/google/protobuf/HBaseZeroCopyByteString.java

```diff
@@ -28,6 +28,28 @@
  * @since 0.96.1
  */
 public final class HBaseZeroCopyByteString  {
+
```

Comment:
This is a bit of a nuanced thing.  It's a class from the HBase API that doesn't work well with newer versions of protubf.  I can explain offline